<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AntennaTrackGadgetOptionsPage</class>
 <widget class="QWidget" name="AntennaTrackGadgetOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>359</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Mode:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="connectionMode"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="serialPage">
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="serialLabel">
            <property name="text">
             <string>Serial Connection</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="serialFormLayout">
            <item row="0" column="1">
             <widget class="QComboBox" name="portComboBox"/>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="portSpeedComboBox"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="dataBitsLabel">
              <property name="text">
               <string>Data Bits:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="stopBitsLabel">
              <property name="text">
               <string>Stop Bits:</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="parityLabel">
              <property name="text">
               <string>Parity:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="timeoutLabel">
              <property name="text">
               <string>Timeout(ms):</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="flowControlComboBox"/>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="dataBitsComboBox"/>
            </item>
            <item row="4" column="1">
             <widget class="QComboBox" name="stopBitsComboBox"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="portLabel">
              <property name="text">
               <string>Port:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="portSpeedLabel">
              <property name="text">
               <string>Port Speed:</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QComboBox" name="parityComboBox"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="flowControlLabel">
              <property name="text">
               <string>Flow Control:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QSpinBox" name="timeoutSpinBox">
              <property name="maximum">
               <number>100000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>connectionMode</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
